TUTORIAL.md

```diff
@@ -227,19 +244,20 @@ node {
 }
 ```
 
-Some environment variables are defined by Jenkins by default, as for freestyle builds.
-For example, `env.BUILD_TAG` can be used to get a tag like `jenkins-projname-1` from Groovy code, or `$BUILD_TAG` can be used from a `sh` script.
+Some environment variables are defined by Jenkins by default.
+
+**Example**: `env.BUILD_TAG` can be used to get a tag like `jenkins-projname-1` from Groovy code, or `$BUILD_TAG` can be used from a `sh` script.
 
-See the help in the _Snippet Generator_ for the `withEnv` step for more details on this topic.
+See  Help in the **Snippet Generator** for the `withEnv` step for more details on this topic.
 
-## Build parameters
+## Configuring to Accept Build Parameters
 
-If you have configured your workflow to accept parameters when it is built (_Build with Parameters_), these will be accessible as Groovy variables of the same name.
+If you have configured your workflow to accept parameters when it is built - **Build with Parameters** - they are accessible as Groovy variables of the same name.
 
-# Recording test results and artifacts
+# Recording Test Results and Artifacts
 
-Rather than failing the build if there are some test failures, we would like Jenkins to record them, but then proceed.
-We would also like to capture the JAR that we built.
+Rather than failing the build if there are test failures, you want Jenkins to record them -- and then proceed.
+You must capture the JAR that you built.
```

Comment:
“must” is misleading. _If_ you want it saved, then you must of course use the archive step.